### PR TITLE
v1.3.0-beta.1 changelog + docs site

### PR DESCRIPTION
**Pull request by skriss**
@carlisia @ashish-amarnath

### CODE REVIEW

File: site/docs/v1.3.0-beta.1/api-types/README.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Table of Contents
+
+## API types
+
+Here we list the API types that have some functionality that you can only configure via json/yaml vs the `velero` cli
+(hooks)
```
Comment (ashish-amarnath) [inline site/docs/v1.3.0-beta.1/api-types/README.md]: "(hooks)" seems out of place

Comment (skriss) [reply site/docs/v1.3.0-beta.1/api-types/README.md]: k, that's existing text in the master/v1.2 docs, so I'm not gonna change it as part of this PR, can do it separately.

Comment (ashish-amarnath) [reply site/docs/v1.3.0-beta.1/api-types/README.md]: Maybe I misunderstand what hooks mean. Does it mean like cli commands?

Comment (ashish-amarnath) [reply site/docs/v1.3.0-beta.1/api-types/README.md]: we can resolve this in a different PR.

**Comment by skriss** [comment]
@ashish-amarnath anything else? just waiting on an approve/merge from you here.

**Pull request merged**